_Week5/Group 3/program.do

```diff
@@ -0,0 +1,19 @@
+cap prog drop regcompare
+prog def regcompare
+	syntax anything
+	local y: word 1 of `anything'
+	local x1: word 2 of `anything'
+	local x2: word 3 of `anything'
+	gen `x1'`x2' = `x1'*`x2'
+	quietly reg `y' `x1'
```

Comment:
One thing to note is that your significance levels change depending on whether your regression is robust (For example, in your first demonstration case, the interaction term would not be significant with robust standard errors). Is there a reason why you chose to run a regular regression instead of a robust one?